packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/BackButton/BackButton.js

```diff
@@ -1,40 +1,47 @@
-import PropTypes from 'prop-types'
-import { Button, Text } from 'grommet'
-import styled from 'styled-components'
-import withThemeContext from '@zooniverse/react-components/helpers/withThemeContext'
-import theme from './theme'
+import { func, object } from 'prop-types'
+import styled, { css } from 'styled-components'
 import { useTranslation } from '@translations/i18n'
+import PrimaryButton from '@zooniverse/react-components/PrimaryButton'
 
-export const StyledBackButtonWrapper = styled.div`
+const StyledButton = styled(PrimaryButton)`
   margin-right: 1ch;
   flex: 0 0 33%;
+
+  ${props =>
+    props.theme.dark
+      ? css`
+          border: white 1px solid;
+          font-color: white;
+          background: ${props.theme.global.colors['dark-1']};
+        `
+      : css`
+          border: none;
+          font-color: black;
+          background: ${props.theme.global.colors['light-1']};
+        `}
+
+  &:focus:not(:disabled),
+  &:hover:not(:disabled) {
+    background: linear-gradient(180deg, #c0e5e7, #84ccd1);
+  }
```

Comment:
This `styled` component replaces all of the code that was imported via `withTheme`.